packages/expo-modules-core/ios/AppDelegates/EXReactNativeFactoryDelegate.h

```diff
@@ -2,11 +2,14 @@
 
 #pragma once
 
-#import <ExpoModulesCore/RCTAppDelegateUmbrella.h>
+#import "RCTDefaultReactNativeFactoryDelegate.h"
```

Comment:
not sure whether this break use_frameworks! build. we should try to import from `#import <React.../*-umbrella.h>`

also not uses double quoted import